Adyen/Assets/ko.lproj/Localizable.strings

```diff
@@ -37,6 +37,7 @@
 "adyen.phoneNumber.placeholder" = "123–456–789";
 "adyen.card.cvcItem.placeholder.digits" = "%@자리";
 "adyen.emailItem.title" = "이메일 주소";
+"adyen.emailItem.placeHolder" = "이메일 주소";
```

Comment:
this line change is on some files but not others 🤔

Reply:
👍